deployment/update_production.sh

```diff
@@ -34,7 +34,7 @@ sudo -H -u $USERNAME git fetch
 # Note that apache should not be running during most of the upgrade,
 # since then e.g. the backup might be incomplete or the code does not
 # match the database layout, or https://github.com/e-valuation/EvaP/issues/1237.
-[[ -z "$GITHUB_WORKFLOW" ]] && sudo service apache2 stop
+[[ -z "$GITHUB_WORKFLOW" ]] && sudo ./deployment/enable_maintenance_mode.sh
```

Comment:
When I checked out the branch, the scripts did not have the executable bit set, so this line would break